R/modules_grammar_hpc.R

```diff
@@ -274,20 +373,21 @@ remove_dead_scuttle.HPCell = function(input_hpc, group_by = NULL, target_input =
           alive_identification(
             empty_tbl,
             annotation_tbl,
-            grouping_column
+            grouping_column,
+            gene_nomenclature = gene_nomenclature
```

Comment:
Same as above, she nomenclature should be dropped from all modules, and use internally the `gene_nomenclature ` target